api/models/rules.py

```diff
@@ -0,0 +1,28 @@
+"""Rule model."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .base import Base
+from .year import Year
+
+
+class Rules(Base):
+    """Stores rules, terms and conditions for year."""
+
+    year = models.ForeignKey(
+        Year,
+        verbose_name=_("Year"),
+    )
+    text = models.TextField(
```

Comment:
Not required, but it's nice to have markdown editor to help you remember the Markdown features. And it is very easy to use.